assumecontrol.lua

```diff
@@ -0,0 +1,64 @@
+--Assume Direct Control of the unit you're viewing. This Can Hurt You.
```

Comment:
A bunch of newer scripts use hyphens to separate words in their names, like `assume-control`. I know they're not all consistent, but for this (and some others), it might be more readable if you do that.

Reply:
Don't like hyphens, and I don't think camelCase fits the styling does it?